crates/biome_cli/tests/snapshots/main_commands_migrate_eslint/migrate_eslintrcjson_missing_biomejson.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: content
+---
+## `.eslintrc.json`
+
+```json
+{}
+```
+
+# Termination Message
+
+```block
+migrate ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Migration has encountered an error: Biome couldn't find the Biome configuration file.
```

Comment:
Should we add a message to suggest running `biome init` first? Or maybe something like:

"If this project has not yet been set up with Biome yet, please following the Getting Started guide first: https://biomejs.dev/guides/getting-started/"

Reply:
I am a bit hesitant for this.
What is the better:
1. asking the user to run `biome init`
2. creating an empty config?

Reply:
The `migrate` command isn't meant to *create* a configuration file. Creating a configuration file would go against what `biome migrate` does.

Reply:
> The migrate command isn't meant to create a configuration file. Creating a configuration file would go against what biome migrate does.

I agree that it doesn't make sense for the main command `biome migrate` to create a config file.
However, I think it makes perfect sense for its subcommands `biome migrate <prettier|eslint>`.

Reply:
> However, I think it makes perfect sense for its subcommands `biome migrate <prettier|eslint>`.

I think this is a good point. It feels natural to initialize a project using a migration instead of the regular `biome init`, but in that case we do need to update the documentation to make sure the Getting Started guide points this out. It also makes sense then to let `biome init` check for the presence of Prettier/ESLint config files and ask whether the user really wants to init with a default config, or if a migration should be performed instead.